README.rst

```diff
@@ -0,0 +1,243 @@
+``pytest-cov``, ``pytest-randomly``, ``pytest-xdist``, ``pytest-html``,
+``pytest-timeout``, ``pytest-durations``
+
+After this installation, you can launch the test suite:
+::
+   pytest hidimstat
+
+Examples
+""""""""
+
+To run the examples, we recommend you install the example dependencies with pip:
+::
+    pip install hidimstat[example]
+
+For running the examples, it's necessary to install Matplotlib >= |MatplotlibMinVersion| and seaborn >=
+|SeabornMinVersion|.
+
+| After this installation, you can run any example in the `examples <https://github.com/mind-inria/hidimstat/tree/main/examples>`_ folder.
+| Or you can download some of them from the `documentation <https://hidimstat.github.io/dev/auto_examples/index.html>`_.
+
+Documentation
+"""""""""""""
+
+The documentation is built with Sphinx. We recommend you install the 
+documentation dependencies with pip:
+::
+    pip install hidimstat[doc]
+
+After this installation, you can build the documentation from the source using 
+the Makefile in doc_conf:
+::
+    make html
+
+Contact us
+----------
+
+.. # Add a way to contact maintainers 
+  TODO this needs to be updated when there is a change of maintainers
+
+Currently, this library is supported by the `INRIA <https://www.inria.fr/en>`_ 
+team `MIND <https://www.inria.fr/fr/mind>`_. |br|
+If you want to report a problem or suggest an enhancement, we would love for you 
+to `open an issue <https://github.com/mind-inria/hidimstat/issues/new>`_ at 
+this GitHub repository so we can address it quickly. |br|
+For less formal discussions or to exchange ideas, you can contact the main 
+contributors:
+
++-------------------+-------------------+-------------------+-------------------+
+|   Lionel Kusch    | Bertrand Thirion  |  Joseph Paillard  | Angel Reyero Lobo |
++-------------------+-------------------+-------------------+-------------------+
+|    |avatar LK|    |    |avatar BT|    |     |avatar JP|   |    |avatar AR|    |
++-------------------+-------------------+-------------------+-------------------+
+
+.. |avatar LK| image:: https://avatars.githubusercontent.com/u/17182418?v=4 
+  :target: https://github.com/lionelkusch
+.. |avatar BT| image:: https://avatars.githubusercontent.com/u/234454?v=4
+  :target: https://github.com/bthirion
+.. |avatar JP| image:: https://avatars.githubusercontent.com/u/56166877?v=4 
+  :target: https://github.com/jpaillard
+.. |avatar AR| image:: https://avatars.githubusercontent.com/u/79699169?v=4 
+  :target: https://github.com/AngelReyero
+
+Citation
+--------
+
+If you use a HiDimStat method for your research, you'll find the associated 
+reference paper in the method description, and we recommend that you cite it.
+
+..
+  TODO add the section for citing the library once a Zenodo repository is made
+  or a paper is published.
+
+If you publish a paper using HiDimStat, please contact us or open an issue! 
+We would love to hear about your work and help you promote it.
+
+License
```

Comment:
I think this section is redundant with both the badge and the about pannel in github